sql/src/test/java/org/apache/druid/quidem/DruidQuidemCommandHandler.java

```diff
@@ -202,10 +220,16 @@ protected final void executeExplain(Context context) throws IOException
         if (node instanceof DruidRel<?>) {
           node = ((DruidRel<?>) node).unwrapLogicalPlan();
         }
-        String str = RelOptUtil.dumpPlan("", node, SqlExplainFormat.TEXT, SqlExplainLevel.EXPPLAN_ATTRIBUTES);
+        String str = getString(node);
         context.echo(ImmutableList.of(str));
       }
     }
+
+    protected String getString(RelNode node)
```

Comment:
note: I feel like  `getString` is a bit too generic - and it could have a better name: `getStringForRel`  ; `dumpRelToString` ; `convertRelToString` ...not that these are that much better...

Reply:
made it `convertRelToString`